Content.Server/Chemistry/ReagentEffects/WearableReaction.cs

```diff
@@ -0,0 +1,31 @@
+using Content.Shared.Inventory;
+using Content.Shared.Chemistry.Reagent;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server.Chemistry.ReagentEffects
+{
+    public sealed partial class WearableReaction : ReagentEffect
```

Comment:
Above this, please add some brief documentation explaining what this does and what it's for. You can incorporate some text from the PR description.